iphone/Classes/TiUISearchBar.m

```diff
@@ -150,6 +150,18 @@ - (void)setColor_:(id)value
   }];
 }
 
+- (void)setFieldBackgroundImage_:(id)arg
+{
+  [[self searchBar] setSearchFieldBackgroundImage:[self loadImage:arg] forState:UIControlStateNormal];
+  [self.searchBar setSearchTextPositionAdjustment:UIOffsetMake(8.0, 0.0)];
```

Comment:
I am not sure the 8px padding should be forced. We could rather expose it as an own property. What do you think?

Reply:
I force it due to [this](http://openradar.appspot.com/radar?id=6667237251350528).
After setting the `fieldBackgroundImage`, the textfield lose its horizontal margin.

Of course can be exposed as an own property, but for this specific case we avoid to force the dev to set it after setting the `fieldBackgroundImage`.

Reply:
I am fine then!